frontends/web/src/routes/device/bitbox02/settings.tsx

```diff
@@ -129,6 +144,9 @@ export const Settings: FunctionComponent<Props> = ({ deviceID }) => {
                     </Footer>
                 </div>
             </div>
+            <Guide>
+                <Entry key="accountDescription" entry={t('guide.bitbox02.rootFingerprint')} />
+            </Guide>
         </div>
     );
 }
```

Comment:
In the comment you mention to move it into account info.

> Now I am thinking though if it might be better to show the fingerprint in the account info along with the xpub. If we show the xpub in descriptor format, the root fingerprint would be part of that anyway

move to account-info?

https://github.com/digitalbitbox/bitbox-wallet-app/pull/1692#discussion_r862725886